Reference/Searching/Examine/Quick-Start/index.md

```diff
@@ -66,208 +103,162 @@ Umbraco ships with three indexes:
 :::
 
-The `searcher` has a CreateQuery method, where you can choose to search content, media or members eg:
+The `Searcher` has a CreateQuery method, where you can choose to search content, media or members eg:
 ```csharp
-searcher.CreateQuery("content")
+Searcher.CreateQuery("content")
 ```
 
 From here you can see how we can chain together the logic to perform the search. In the example we are searching all `content` using the `person` Document Type, where the `nodeName` is equal to the search term that was typed in the input bar.
 
 ```csharp
-searcher.CreateQuery("content").NodeTypeAlias("person").And().Field("nodeName", searchTerm)
+Searcher.CreateQuery("content").NodeTypeAlias("person").And().Field("nodeName", searchTerm)
 ```
-### Execute the Search
 
-Finally calling `.Execute()` at the end of the query logic will trigger the search and return a set of matching search results that we can loop through.
+We are calling `.Execute()` at the end of the query logic will trigger the search and return a set of matching search results that we can loop through.
+We can then select the ids.
+## Using the umbraco helper to get content
+We want to use the UmbracoHelper, so we can use the id to get the actual content.
+Lets inject the umbraco helper into our service!
+```csharp
+private readonly IExamineManager _examineManager;
+private readonly UmbracoHelper _umbracoHelper;
 
-The final template looks like this:
+public SearchService(IExamineManager examineManager, UmbracoHelper umbracoHelper)
+{
+    _examineManager = examineManager;
+    _umbracoHelper = umbracoHelper;
+}
+```
+
+The final method looks like this:
 ```csharp
-@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<People>
-@using Microsoft.AspNetCore.Mvc.TagHelpers
-@using Examine
-@inject IExamineManager ExamineManager;
-@{
-    Layout = "master.cshtml";
+public IEnumerable<IPublishedContent> SearchContentNames(string query)
+{
+    IEnumerable<string> ids = Array.Empty<string>();
+    if (!string.IsNullOrEmpty(query) && _examineManager.TryGetIndex("ExternalIndex", out IIndex? index))
+    {
+        ids = index.Searcher.CreateQuery("content").NodeTypeAlias("person").And().Field("nodeName", query).Execute()
+            .Select(x => x.Id);
+    }
+
+    foreach (var id in ids)
+    {
+        yield return _umbracoHelper.Content(id);
+    }
 }
-@{
-    void SocialLink(string content, string service)
+```
+After getting the ids from our search, we then loop through the list, and return the content.
+
+# Creating a custom viewmodel
+We will now need a custom view model, so that we can pass it to the view.
+```csharp
+using Umbraco.Cms.Core.Models.PublishedContent;
+
+namespace Umbraco.Cms.Web.UI.Models;
+
+public class SearchViewModel : PublishedContentWrapped
```

Comment:
Perhaps explain why it's necessary to use `PublishedContentWrapped` as base class for the view model?